assessment2.js

```diff
@@ -0,0 +1,148 @@
+                var info = document.createElement('p');
+                info.setAttribute('class', `${i}`)
+                info.innerHTML = movieList[i].description
+                para.appendChild(info)
+
+                movies.push(movie);
+                
+                var option = document.createElement('option');
+                option.text = movieList[i].title
+                select.add(option);
+                
+            }
+            
+            console.log(movies)
+
+
+            // {
+            //     var name = document.createElement('h3');
+            //     name.setAttribute('class', `${i}`)
+            //     var year = document.createElement('p');
+            //     year.setAttribute('class', `${i}`)
+            //     var info = document.createElement('p');
+            //     info.setAttribute('class', `${i}`)
+            //     // console.log(i)
+            //     // console.log(name.innerText = movieList[i].title);
+            //     // console.log(year.innerHTML = movieList[i].release_date);
+            //     // console.log(info.innerHTML = movieList[i].description);
+            //   var option = document.createElement('option');
+            //  option.text = movieList[i].title
+            //     // console.log(option)
+            //     select.add(option);
+            // }
+
+        } catch (err) {
+            console.log('oops, there was an error. please try again', err);
+        }
+    })
+
+
+}
+
+const printInfo = () => {
+    let changeFilm = document.querySelector('films');
+changeFilm.addEventListener('change', (event) => {
+    let result = document.querySelector('.1')
+    console.log(result)
+})
+} 
+
+//         switch (movies) {
+//             case movies[0]: console.log('1');
+//                 break;
+
+//             case movies[1]: console.log('2');
+//                 break;
+
+//             default:
+//                 console.log('ehh')
+
+//         }
+//     })
+// }
+
+
+
+
+// const addReview = () => {
+// let reviewList = document.querySelector('ul');
+
+//   let reviewText = document.getElementById('review').value.trim()
+//   if (reviewText) {
+//     let newReview = document.createElement('li')
+//     newReview.innerText = reviewText
+//     console.log(newReview)
+//     reviewList.appendChild(newReview)
+//     document.getElementById('review').value = '' // clears the textbox after button is submitted
+//   } else {
+//    console.log('Error. The list cannot be empty');
+//   }
+// }
+
+
+const formListener = () => {
+    let form = document.querySelector('form');
+    form.addEventListener("submit", event => {
+        event.preventDefault()
+    });
+    sendReview()
+
+}
+
+const sendReview = () => {
+    let reviewInput = document.querySelector('#review').value;
+    let reviewList = document.querySelector('#review-list');
+    if(reviewInput){
+        let newReview = document.createElement('li')
+    newReview.innerText = reviewInput.value;
```

Comment:
This function is ALMOST functional. You have to remove "value" after reviewInput. Here is the fix: 
newReview.InnerText = reviewInput 
After moving sendReview function into the event listener, your review functionality should work. Very close